src/components/ui/Hero/hero.scss

```diff
@@ -7,28 +7,37 @@
   background-color: var(--bg-secondary-default);
 
   @include media(">=notebook") {
+    position: relative;
     flex-direction: row;
+    justify-content: flex-end;
   }
 
   [data-hero-content] {
     width: 100%;
-    padding: var(--space-6) var(--space-4);
+    padding: var(--space-5) 0 var(--space-6);
     color: var(--color-text-inverse);
 
     @include media(">=notebook") {
-      padding: var(--space-13) var(--space-9) var(--space-13) 0;
+      position: absolute;
+      left: 0;
+      width: 100%;
+      height: 100%;
+      padding: var(--space-13) 0;
     }
 
     .banner-info-content {
       display: flex;
       flex-direction: column;
       justify-content: space-between;
-      max-width: 100%;
       height: 100%;
-      margin-left: auto;
 
       @include media(">=notebook") {
-        max-width: 30rem;
+        padding-left: var(--space-5);
+
+        > div {
+          max-width: 42%;
```

Comment:
I guess, we might need to tweak these numbers in the future, but for now it's okay :)

Reply:
🤡 :mild-panic: